_posts/2018-04-11-dont-make-users-think-about-updating-software.md

```diff
@@ -11,35 +11,35 @@ tags:
   - update-service
 ---
 
-As software developers, we want the people using the software we create to ~~enjoy~~ not _hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, many hours are spent debating and designing user interactions that are simple, effective, and intuitive.
+As software developers, we want the people who are using our software to ~~enjoy~~ _not hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, we spend many hours designing, debating, and redesigning to ensure that user interactions with our software are simple, effective, and intuitive.
 
 ### skip to the [tl;dr <i class="fas fa-long-arrow-alt-down"></i>](#tldr)
 
-> The people that are using your software are motivated by the desire to save time
+> The people who are using your software are motivated by the desire to save time.
 
-Building on the blog post from Scott Davis, _[Cache breaking for faster web application updates]({{site.baseurl}}{% post_url 2018-03-23-cache-breaking-for-faster-web-application-updates %})_, which is about website caching techniques for delivering faster updates to users, at it's core is about improving the user experience of website applications. Getting updates to users is an often overlooked feature and an after thought. But it is very important—_Especially when you ship a lot of bugs_.
+Building on the blog post from Scott Davis, _[Cache Breaking for Faster Web Application Updates]({{site.baseurl}}{% post_url 2018-03-23-cache-breaking-for-faster-web-application-updates %})_, which is about website caching techniques for delivering faster updates to users, at its core software development is about improving the user experience of website applications. Getting updates to users is an often overlooked feature and an afterthought. But it is very important—_especially when you ship a lot of bugs_.
```

Comment:
Hmm...might need a comma after "core"...Also, did I understand that correctly? "at its core, software development is about..."?

Reply:
yeah i wasn't sure, it seemed like there was a lot of commas in this sentence. I thought you might add some dashes and other stuff! I was really curious to see what you did here...

So the at it's core is referencing scott's posts content not software development in general.